include/dkm_parallel.hpp

```diff
@@ -15,6 +15,11 @@
 #include <vector>
 
 #include "dkm.hpp"
+#include "dkm_utils.hpp""
```

Comment:
please remove the extra `"` token from the end of the `#include "dkm_utils.hpp""`